packages/pds-ember/app/styles/pds/components/index.scss

```diff
@@ -19,4 +19,5 @@
 @use "select";
 @use "sidebar";
 @use "skip-link"; // TODO: move to "link/skip-link" as variant
```

Comment:
Is there either a design spec for this edge case, or an existing JIRA ticket to reference?

My concern would be that this important question could get lost, if it isn't captured elsewhere.

Reply:
captured via PDS-195